app/pods/publications/publication/documents/controller.js

```diff
@@ -232,6 +246,7 @@ export default class PublicationDocumentsController extends Controller {
   @action
   cancelTranslationModal() {
     set(this.translateActivity, 'mailContent', '');
+    set(this.translateActivity, 'mailSubject', '');
     this.showTranslationModal = false;
```

Comment:
Niet perse dit ticket, maar de state van finalTranslationDate en pieces, moet die ook worden reset hier ?

Reply:
Ja dat lijkt me wel ja.

Reply:
Maar dat zit al ergens in een andere branch dus ik hou het hier atomair